lightning/src/events/mod.rs

```diff
@@ -1141,6 +1200,55 @@ pub enum Event {
 	///
 	/// [`ChannelHandshakeConfig::negotiate_anchors_zero_fee_htlc_tx`]: crate::util::config::ChannelHandshakeConfig::negotiate_anchors_zero_fee_htlc_tx
 	BumpTransaction(BumpTransactionEvent),
+	#[cfg(any(splicing, dual_funding))]
```

Comment:
```suggestion
	#[cfg(any(dual_funding, splicing))]
```
Let's stick to this order for consistency (easier to search, btw alphabetical order)